src/status_im/subs.cljs

```diff
@@ -89,6 +89,7 @@
 (reg-root-key-sub :get-pairing-installations :pairing/installations)
 (reg-root-key-sub :network/type :network/type)
 (reg-root-key-sub :tooltips :tooltips)
+(reg-root-key-sub :get-supported-biometric-auth :supported-biometric-auth)
```

Comment:
don't add the `get-` prefix here it's superfluous, the other subs that use that are wrong